lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -1069,6 +1506,42 @@ uint64_t PatchEntryPointMutate::pushFixedShaderArgTys(SmallVectorImpl<Type *> &a
   return inRegMask;
 }
 
+// =====================================================================================================================
+// Get UserDataUsage struct for the merged shader stage that contains the given shader stage
+//
+// @param stage : Shader stage
+PatchEntryPointMutate::UserDataUsage *PatchEntryPointMutate::getUserDataUsage(ShaderStage stage) {
+  stage = getMergedShaderStage(stage);
+  m_userDataUsage.resize(std::max(m_userDataUsage.size(), static_cast<size_t>(stage) + 1));
+  if (!m_userDataUsage[stage])
+    m_userDataUsage[stage] = std::make_unique<UserDataUsage>();
+  return &*m_userDataUsage[stage];
+}
+
+// =====================================================================================================================
+// Get the shader stage that the given shader stage is merged into.
+// For GFX9+:
+// VS -> TCS (if it exists)
+// VS -> GS (if it exists)
+// TES -> GS (if it exists)
```

Comment:
Add a @param comment, describe the input stage is the original stage before merging.